mpmp/src/model/GameState.java

```diff
@@ -7,11 +7,10 @@
 
 	public static void startGame() {
 		state = RunningGame;
+		System.out.println("The game is now running");
 	}
 
 	public static boolean running() {
-		if (state == RunningGame)
-			return true;
-		return false;
+		return state == RunningGame;
```

Comment:
I'm sorry for the atypical verbosity I've displayed in these three lines. After all, I am known for terse code. Maybe I've been exposed to *too much Java*. :smile: